google/cloud/forseti/common/gcp_api/admin_directory.py

```diff
@@ -27,9 +27,12 @@
 
 API_NAME = 'admin'
 
+GROUP_SETTINGS_API_NAME = "groupssettings"
```

Comment:
All the group settings API code needs to be in a separate file, there needs to be only one API client per python module in this directory.

Reply:
Please do as Andrew suggested.  This is also why it's a good idea to get the one-pager implementation plan in place, before plunging ahead with the coding.